libs/sdk-common/src/liquid/bip21.rs

```diff
@@ -0,0 +1,174 @@
+use elements::{
+    address::{Address, AddressError, AddressParams},
+    hashes::hex::HexToArrayError,
+    issuance::AssetId,
+};
+use serde::Serialize;
+use std::{num::ParseFloatError, str::FromStr, string::FromUtf8Error};
+use urlencoding::decode;
+
+use crate::{
+    invoice::{parse_invoice, InvoiceError},
+    prelude::Network,
+};
+
+#[derive(Debug, Clone, Serialize)]
+pub struct LiquidAddressData {
```

Comment:
NIT: LiquidBit21 ? as it is not only an address.

Reply:
I see now that you are using it for plain liquid address also so perhaps that's fine. @hydra-yse ?